packages/commonwealth/client/scripts/views/pages/view_thread/ViewThreadPage.tsx

```diff
@@ -236,6 +236,14 @@ const ViewThreadPage = ({ identifier }: ViewThreadPageProps) => {
     setAreProfilesLoaded(true);
   }, [comments, thread, areProfilesLoaded]);
 
+  useEffect(() => {
+    if (thread && thread.title) {
+      document.title = `${app.chain.meta.name} – ${thread.title}`;
+    } else {
+      document.title = `${app.chain.meta.name} – View thread`;
+    }
+  }, [thread]);
+
```

Comment:
why not useManageDocumentTitle hook here?

```
const documentTitle = thread?.title || 'View thread';
useManageDocumentTitle(`${app.chain.meta.name} – ${documentTitle}`)
```

Reply:
I tested this solution but thread only gets a non null value on the component render.
This way 'View thread' is always set as the title.

For this case we would have to change the hook just for the sake of the View Thread page.

Thoughts?

Reply:
it will work if you fix this 👇 

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/3bfd6a4d-6f20-478e-b983-749536bb79b9)

This is an example that these warning are not useless thing. They point out bug prone places